onnxscript/function_libs/torch_aten/ops/core.py

```diff
@@ -2321,7 +2321,7 @@ def aten_flip(self: TTensor, dims: INT64) -> TTensor:
     neg_1 = op.Constant(value_int=-1)
     starts = op.Expand(neg_1, shape_dim)  # something like [-1, -1, -1]
     steps = op.Expand(neg_1, shape_dim)  # something like [-1, -1, -1]
-    ends = starts * 65535  # something like [-65535, -65535, -65535]
+    ends = starts * _INT8_MIN  # something like [-127, -127, -127]
     result = op.Slice(self, starts, ends, dims, steps)
```

Comment:
Ends in start is defined as `ends (heterogeneous) - Tind: 1-D tensor of ending indices (exclusive) of corresponding axis in axes` and `Tind ` is in ( tensor(int32), tensor(int64) ). https://onnx.ai/onnx/operators/onnx__Slice.html

I think we should just default to use int64 in all cases